pkg/controller/serviceaccount/config/types.go

```diff
@@ -27,4 +29,7 @@ type SAControllerConfiguration struct {
 	// rootCAFile is the root certificate authority will be included in service
 	// account's token secret. This must be a valid PEM-encoded CA bundle.
 	RootCAFile string
+	// legacyTokenCleanUpPeriod is the period of time since the last usage of an
+	// auto-generated service account token before it can be deleted.
+	LegacyTokenCleanUpPeriod time.Duration
```

Comment:
do we intend this cleanup to be able to be disabled once the feature gate graduates? if so, how would someone disable it? normally, we'd have a separate controller they could disable with `--controllers=-serviceaccount-legacy-tokencleaner` or something

Reply:
I just found there is an existing tokencleaner which is disabled by default. Should I reuse this one and make some revisions on that or I'd better create a new serviceaccount-legacy-tokencleaner?

Reply:
Discussed with @zshihang. Can I just make the time of LegacyTokenCleanUpPeriod as a pointer and thus if there is no input for the flag, the value will be nil (which will be then defaulted to 366 * 24 h). But if we specifically input via --legacy-token-clean-up-period=0, then in the token_controller.go, we can disable the cleaning up if we find this value is 0.